WireAPI/Sources/WireAPI/Models/UpdateEvent/ConversationEvent/ConversationMLSMessageAddEvent.swift

```diff
@@ -42,4 +57,13 @@ public struct ConversationMLSMessageAddEvent: Equatable, Codable {
 
     public let message: String
 
+    /// The date the message was received.
+
+    public let timestamp: Date?
+
+    /// The decrypted current message + decrypted buffered messages
+    /// along with the related sender client ID for each message.
+
+    public var decryptedMessages: [DecryptedMessage] = []
```

Comment:
I find it a bit confusing to name the type `DecryptedMessage`, having a property `message` as well as another property named `decryptedMessages`.

IS it a message, 
HAS it A message or 
HAS it MANY messages?